### ISSUE TITLE
Humerus fracture

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5885]
**Preferred term label:** Humerus fracture

**Synonyms** Broken humerus; Fracture of the humerus

**Definition (free text, please give PubMed ID)** Fracture of the humerus.

**Parent term (use hpo.jax.org/app)** Abnormality of the humerus HP:0003063; Bone fracture HP:0020110; Fractures of the long bones HP:0003084

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@JillMiller we have just recently added a bunch of fracture terms. They are visible on the development server
https://ols.monarchinitiative.org/ontologies/hp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FHP_0041055

The terms will be visible on the main HPO page after the next release in August.

I will add these synonyms.
